recipe/laravel.php

```diff
@@ -35,6 +35,16 @@
     'storage/logs',
 ]);
 
+set('laravel_version', function () {
+    $result = run('{{bin/php}} {{release_path}}/artisan --version');
+
+    preg_match_all('/version\ (.+?)$/', $result, $matches);
+
+    $version = $matches[1][0] ?? 5.4;
+
+    return (float) $version;
```

Comment:
See the docs on http://php.net/manual/en/function.version-compare.php
That is specifically created to handle those situations.